### PR TITLE
fix(social-btn): add white border on dark background for active state

### PR DESCRIPTION
**Pull request by hannahiss**
on dark background, need a white border on active tate so that the size of button doesn't decrease when active (report form branch main-his-social-border-on-dark)

### CODE REVIEW

**Comment by hannahiss** [comment]
Linked to #1121

**Comment by louismaximepiton** [comment]
Just to remind what we've said in private : 
The `.active` should be back-ported to v4, since this class is actually brought by the `button-variant` mixin.

**Comment by hannahiss** [comment]
> Just to remind what we've said in private : The `.active` should be back-ported to v4, since this class is actually brought by the `button-variant` mixin.

You can review. Thanks

File: scss/_buttons.scss [hunk 1]
```diff
@@ -190,14 +190,17 @@ fieldset:disabled a.btn {
     }
 
     &,
-    &.btn-inverse:active {
+    &.btn-inverse:active,
+    &.btn-inverse.active {
       background-color: $white;
       background-image: escape-svg(url("data:image/svg+xml,#{$icon}"));
+      border-color: $white;
```
Comment (louismaximepiton) [inline scss/_buttons.scss:197]: This line should maybe be in another selector since it breaks the normal social buttons

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
:warning: Normal social buttons are broken ⚠️

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**